src/Microsoft.Identity.Web.TokenAcquisition/ClaimConstants.cs

```diff
@@ -99,5 +99,15 @@ public static class ClaimConstants
         /// Name Identifier ID claim: "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier".
         /// </summary>
         public const string NameIdentifierId = "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier";
+
+        /// <summary>
+        /// Username claims for ROPC flow.
+        /// </summary>
+        public const string Username = "xms-username";
```

Comment:
Please resolve the warnings in the GitHub Actions. You can see them in the UI via GitHub, or in the IDE In VS.